spec/index.bs

```diff
@@ -4898,12 +4898,14 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
                     </dl>
                 ::
                     If |entry|.{{GPUBindGroupLayoutEntry/buffer}}.{{GPUBufferBindingLayout/minBindingSize}} is not `0`:
-                    - If the last field of the corresponding structure defined in the shader has an unbounded array type,
+                    - If the store type of the corresponding buffer variable in the shader is a structure type, and the
```

Comment:
I would have expected this to be split into two cases: one where the runtime sized array is the store type and another where it is the last member of a structure that is the store type. The "otherwise" case seems to only cover non-runtime-sized types.